waku/node/waku_node.nim

```diff
@@ -575,6 +575,7 @@ proc filterSubscribe*(node: WakuNode,
   if pubsubTopic.isSome():
     info "registering filter subscription to content", pubsubTopic=pubsubTopic.get(), contentTopics=contentTopics, peer=remotePeer.peerId
 
+    let contentTopics = contentTopics.mapIt($it)
```

Comment:
What is this for?

Reply:
It seems to be meant to convert the seq to a string seq but seems not to be necessary

Reply:
I just realised that is wrong, that doesn't work.

The idea is the parameter `contentTopics`  is of type `ContentTopic | seq[ContentTopic]`. Given the expected type for `subscribe` is `seq[ContentTopic]`, that wraps the `ContentTopic` inside a `seq`, to comply with it.

Fixing it.